tempesta_fw/http.c

```diff
@@ -2174,6 +2256,42 @@ tfw_srv_client_block(TfwHttpReq *req, int status, const char *msg)
 					req, status, msg);
 }
 
+static inline bool
+tfw_http_check_wildcard_status(const char c, int *out)
+{
+	switch (c) {
+	case '1':
+		*out = HTTP_STATUS_1XX;
+		break;
+	case '2':
+		*out = HTTP_STATUS_2XX;
+		break;
+	case '3':
+		*out = HTTP_STATUS_3XX;
+		break;
+	case '4':
+		*out = HTTP_STATUS_4XX;
+		break;
+	case '5':
+		*out = HTTP_STATUS_5XX;
+		break;
+	default:
+		return false;
+	}
+	return true;
+}
+
+static inline void
+tfw_http_drop_hmonitor_resp(TfwHttpReq *req, TfwHttpResp *resp)
+{
+	tfw_connection_unlink_msg(resp->conn);
+	tfw_apm_update(((TfwServer *)resp->conn->peer)->apmref,
+		       resp->jrxtstamp,
+		       resp->jrxtstamp - req->jtxtstamp);
```

Comment:
This line fits perfectly in the preceding line of code.

Reply:
Corrected.